tests/FunctionalTestCase.php

```diff
@@ -519,6 +522,25 @@ protected function skipIfTransactionsAreNotSupported(): void
         }
     }
 
+    protected function isEnterprise(): bool
```

Comment:
I recovered this function removed by [#1072](https://github.com/mongodb/mongo-php-library/pull/1072/files#diff-93b2fb11cb160b931a627169b6f5cb9c5d12b940e5a7ff1aec827f39abf1fc11L400-L412)